test/Npgsql.Tests/PoolTests.cs

```diff
@@ -405,15 +469,14 @@ public void ExercisePool(int maxPoolSize, int numTasks, int seconds, bool async)
 
         volatile int StopFlag;
 
-        void AssertPoolState(ConnectorPool? pool, int idle, int busy, int waiting=0)
+        void AssertPoolState(ConnectorPool? pool, int open, int idle)
```

Comment:
Also, let's make the parameters optional and specify parameter names on invocation (e.g. `AssertPoolState(..., open: 1, idle: 2)`) for better clarity. Every time I look at these tests I have to find out what each parameter means.

Reply:
Yes that's really much better for these arguments, great idea.